recipes/opensubdiv/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+import functools
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class OpenSubdivConan(ConanFile):
+    name = "opensubdiv"
+    license = "Modified-Apache-2.0"
```

Comment:
I was just working on writing docs for this #11720...

This is not an SPDX identifier, would you might trying the instructions?

Reply:
Thanks for adding this to the docs, I was not aware of this. Since it's a custom license, I changed it to "LicenseRef-LICENSE.txt"